indexers/keyvalue/RedisDBIndexer/manifest.yml

```diff
@@ -1,12 +1,12 @@
 manifest_version: 1
 name: RedisDBIndexer
 kind: indexer
-description: Use Redis as a key-value indexer
+description: Key-value indexer wrapper around Redis
 author: Jina AI Dev-Team (dev-team@jina.ai)
 url: https://jina.ai
 vendor: Jina AI Limited
 documentation: https://github.com/jina-ai/jina-hub
-version: 0.0.9
+version: 0.0.10
 license: apache-2.0
 keywords: [database, redis]
```

Comment:
maybe add KVindexer or indexer to the keywords?

Reply:
not necessary. We have this information from the file structure.